overlord/ifacestate/ifacemgr.go

```diff
@@ -59,3 +114,43 @@ func (m *InterfaceManager) Ensure() error {
 func (m *InterfaceManager) Stop() error {
 	return nil
 }
+
+// Interfaces returns object holding a lists of all the plugs and slots and their connections.
+func (m *InterfaceManager) Interfaces() *interfaces.Interfaces {
```

Comment:
(note for the future: if all we wanted was to export the methods of the repository, we might instead simply expose the repository, right?)